.github/workflows/tests.yml

```diff
@@ -131,6 +131,16 @@ jobs:
           sudo rm -rf /opt/ghc
           df -h /
 
+      - name: Reclaim some memory
+        run: |
+          set -eux
+
+          free -mt
+          sudo systemctl stop dpkg-db-backup.timer e2scrub_all.timer fstrim.timer logrotate.timer man-db.timer motd-news.timer phpsessionclean.timer update-notifier-download.timer update-notifier-motd.timer
```

Comment:
Just a nit but the commit message seems to have some rebase leftovers?

Reply:
For a permanent machine, I usually opt to uninstall the bits that are not needed like this `php` version but since those are ephemeral, simply turning off the unwanted services is faster (less IO needed). Granted, the differences are not that significant :)

I've removed the rebase leftovers, thanks!

Reply:
All right, thanks.